windows/RNTrackPlayer/Logic/MediaManager.cs

```diff
@@ -1,28 +1,23 @@
-using Newtonsoft.Json.Linq;
-using ReactNative.Bridge;
-using ReactNative.Modules.Core;
+using Microsoft.ReactNative.Managed;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
```

Comment:
i'm curious why do you need this objectmodel using?

Reply:
I'm not sure how they got in there. Just removed them. Thanks.